src/ui/pms/service.go

```diff
@@ -12,9 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package project
+package pms
 
 // PMS is the project mamagement service which abstracts
 // the operations related to projects
 type PMS interface {
+	IsPublic(projectIDOrName interface{}) bool
```

Comment:
PMS should just return data, it doesn't need to check permission, permission should be checked/interpreted by context.